CodePushPackage.m

```diff
@@ -285,6 +285,11 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
                 [CodePushPackage copyEntriesInFolder:unzippedFolderPath
                                           destFolder:newPackageFolderPath
                                                error:&error];
+                if (error) {
```

Comment:
I pushed a change to always delete the folder at the destination before unzipping so that leftover folders will not cause issues.